build-settings/rollup.config.js

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable import/no-commonjs */
 import fs from "fs";
 import autoExternal from "rollup-plugin-auto-external";
 import babel from "rollup-plugin-babel";
 
-const {presets, plugins} = require("./babel.config.js");
+const {presets, plugins} = require("../babel.config.js");
```

Comment:
Why wasn't the babel config moved too?

Reply:
I was worried about it no longer being visible to those things need, but maybe that concern is unfounded since the the things that need are already in this folder.